### PR TITLE
Add event handler

### PR DESCRIPTION
**Pull request by mokejp**
ref.  https://github.com/line/line-bot-sdk-go/issues/7, https://github.com/line/line-bot-sdk-go/issues/15, https://github.com/line/line-bot-sdk-go/pull/9

Add event handler. And support for appengine.

I would like to carefully think about this feature.

- Is it decrease versatility?
- Is it easy to use?
- Is it not better to divide Handlers for each `EventType`?

etc...

### CODE REVIEW

**Comment by sugyan** [comment]
Hmm, I think that this is pretty good.
But, I feel a little cumbersome that every `EventHandlerFunc ` need arguments `Context` and `Client`..

**Comment by mokejp** [comment]
hmm... I was wrong.

This interface cannot use this case: Put more than one event in the queue.
I have to think once more. And maybe your way is better.

File: linebot/httphandler/httphandler.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+func New(channelSecret, channelToken string) (*WebhookHandler, error) {
+	h := &WebhookHandler{
+		channelSecret: channelSecret,
+		channelToken:  channelToken,
+	}
+	return h, nil
+}
+
+// HandleEvents method
+func (eh *WebhookHandler) HandleEvents(f EventsHandlerFunc) {
```
Comment (sugyan) [inline linebot/httphandler/httphandler.go]: `(wh *WebhookHandler)` is better

**Comment by sugyan** [comment]
LGTM! Thanks!!

**Pull request merged**